AutoLoads/MusicManager.gd

```diff
@@ -58,12 +112,10 @@ func load_settings() -> Dictionary:
 	
 func set_volume(volume: float) -> void:
 	volume = clamp(volume, 0.0, 100.0)
-	
 	var volume_db = linear_to_db(volume / 100.0)
 	AudioServer.set_bus_volume_db(0, volume_db)
-		
-	#save_volume(volume)
-
+	save_volume(volume)
```

Comment:
Šitą eilutę užkomentavau nes šitas metodas kviečiamas dafigą kartų per sekundę kai dragini inputą ir gaunasi labai daug skaitymo ir rašymo iš failo. Geriau saugoti failą prijungus `save_volume` prie inputo drag_ended signalo:

![image](https://github.com/user-attachments/assets/eae1182c-4d77-4bbe-8a76-c7c3d32dc36d)